pluginapi/src/main/java/org/robolectric/pluginapi/Sdk.java

```diff
@@ -118,7 +118,6 @@ public int compareTo(@Nonnull Sdk o) {
    * throw org.junit.AssumptionViolatedException to just skip execution of tests on the SDK, with a
    * warning, or throw a RuntimeException to fail the test.
    *
-   * @param testClassName
    */
   public abstract void verifySupportedSdk(String testClassName);
```

Comment:
IMO, this API is public and released to developers many years. It's safe to remove it, because we don't know whether there are developers are using it. We must be curious about public API to ensure API stability.